### PR TITLE
Mark all clinic headers as generated

### PR DESCRIPTION
**Pull request by erlend-aasland**
Automerge-Triggered-By: GH:zware

### CODE REVIEW

**Comment by erlend-aasland** [comment]
The current pattern fails to catch `Modules/_sqlite/clinic/*.h`

File: .gitattributes [hunk 1]
```diff
@@ -42,10 +42,7 @@ PC/readme.txt text eol=crlf
 
 # Generated files
 # https://github.com/github/linguist#generated-code
```
Comment (JelleZijlstra) [inline .gitattributes]:
```suggestion
# https://github.com/github/linguist/blob/master/docs/overrides.md
```
The current link is broken.

**Review by kumaraditya303 — APPROVED** [review]

File: .gitattributes (hunk 1 above)
Comment (erlend-aasland) [reply .gitattributes]: Well observed

**Review by JelleZijlstra — APPROVED** [review]

**Review by zware — APPROVED** [review]

**Comment by erlend-aasland** [comment]
BTW, we should backport this to 3.10 and 3.9. Agreed, @zware?

**Pull request merged**